de.ient.RDPlot.yaml

```diff
@@ -0,0 +1,162 @@
+app-id: de.ient.RDPlot
+runtime: org.kde.Platform
+runtime-version: '5.10'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.gfortran-62
+command: rdplot
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+build-options:
+  append-path: "/usr/lib/sdk/gfortran-62/bin"
+modules:
+  - name: gfortran
+    buildsystem: simple
+    build-commands:
+    - "/usr/lib/sdk/gfortran-62/install.sh"
+
+  - name: lapack
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - "-DCMAKE_BUILD_TYPE=Release"
+    - "-DBUILD_SHARED_LIBS=ON"
+    - "-DBUILD_TESTING=OFF"
+    - "-DCMAKE_Fortran_COMPILER=/usr/lib/sdk/gfortran-62/bin/gfortran"
+    - "-DLAPACKE=ON"
+    - "-DCBLAS=ON"
+    sources:
+    - type: archive
+      url: http://www.netlib.org/lapack/lapack-3.8.0.tar.gz
+      sha512: 17786cb7306fccdc9b4a242de7f64fc261ebe6a10b6ec55f519deb4cb673cb137e8742aa5698fd2dc52f1cd56d3bd116af3f593a01dcf6770c4dcc86c50b2a7f
+    cleanup:
+    - "/lib/cmake"
+  - name: python3-numpy
+    buildsystem: simple
+    build-commands:
+    - python3 setup.py build -j 0
```

Comment:
`-j 0` should probably be `-j $FLATPAK_BUILDER_N_JOBS`

Reply:
thx, changed it. can't do the same for scipy, apperently it cannot be build multi-core

Reply:
Alright, was worth a shot anyway.